src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -175,3 +190,23 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('identity assign', 'iot_central_app_assign_identity')
         g.custom_command('identity remove', 'iot_central_app_remove_identity')
         g.custom_show_command('identity show', 'iot_central_app_show_identity')
+        # g.custom_command('private-endpoint-connection delete', '')
+        # g.custom_command('private-endpoint-connection show', '')
+        # g.custom_command('private-endpoint-connection approve', '')
+        # g.custom_command('private-endpoint-connection reject', '')
+    with self.command_group('iot central app private-endpoint-connection', private_endpoint_sdk,
+                            is_preview=True,
+                            resource_type=ResourceType.MGMT_IOTCENTRAL, min_api='2021-11-01-preview') as g:  # removed param: custom_command_type=private_endpoint_custom_type, 
+        from ._validators import validate_private_endpoint_connection_id
+        g.command('delete', 'delete', confirmation=True, validator=validate_private_endpoint_connection_id)
+        g.show_command('show', 'get', validator=validate_private_endpoint_connection_id)
+        g.custom_command('approve', 'approve_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.custom_command('reject', 'reject_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+    
+    with self.command_group('iot central app private-link-resource', private_link_resource_sdk,
+                            resource_type=ResourceType.MGMT_IOTCENTRAL) as g:
+        from azure.cli.core.commands.transform import gen_dict_to_list_transform
+        g.command('list', 'list_private_link_resources', is_preview=True, min_api='2021-11-01-preview',
+                  transform=gen_dict_to_list_transform(key="value"))
```

Comment:
At present, this code has the following errors, resulting in `iot central app private-endpoint-connection` and `iot central app private-link-resource` can not being loaded normally:

1. The python SDK package `azure-mgmt-iotcentral` corresponding to `ResourceType.MGMT_IOTCENTRAL` does not support multiple api-version folders, so please remove the declaration of `min_api='2021-11-01-preview'` and the declaration of `ResourceType.MGMT_IOTCENTRAL: '2018-09-01'` in the [`_shared.py`](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L240) file


2. There is no method `list_private_link_resources()` in `_private_links_operations.py`, but only method `list()`
[code link](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/iothub/azure-mgmt-iotcentral/azure/mgmt/iotcentral/operations/_private_links_operations.py#L180)